packages/client/src/createClient.test.ts

```diff
@@ -62,7 +62,7 @@ describe(createClient, () => {
       ontologyRid: "",
       userAgent: "",
     };
-    const stack = "foo.bar";
+    const stack = "https://foo.bar";
```

Comment:
Why do we need to add the protocol now?

Reply:
Because when you are doing a local dev, you may not be doing HTTPs so we are just going to force it to be configured properly from the onset